### DISCUSSION TITLE
Can we have langchain llm caching?

### DISCUSSION THREAD

**Discussion by OnlinePage** [msoedov/langcorn#24] [Q&A]
Hi how can we implement it with llm caching either mongodb or sqlite , which serves uniquly to every user with caching to reduce api calls?

**Comment by msoedov** [comment]
Good question. There is a doc for caching https://python.langchain.com/docs/modules/model_io/models/llms/how_to/llm_caching.
You can add to prompt user_id to ensure that uniquely saved.

**Reply by OnlinePage** [reply]
can you give me a minimal example of it 🙂? Actually i know the langchain caching in general ,  but with langcorn to serve the caching with memory to every user uniquly is making me bit confused.
Pls a minimal example...

**Reply by OnlinePage** [reply]
@msoedov please can you provide an simple or  minimal example for  serving the cache with memory to every user uniquely 🥹
